src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -159,11 +143,39 @@ private static void setupLogger(Application application) {
         application.addComponentMenuElement(Logger.class, "OFF logging", (ActionEvent e) -> logger.setLevel(Level.OFF));
     }
 
+    private static void setupFeatures(Application application) {
```

Comment:
Maybe rename to setupFeaturesMenu, not to confuse with features setup by ApplicationManager?